### PR TITLE
Fixed #933: Removed redundant code for the creation of feature relati…

### LINKED ISSUE #933: Plug-In "de.ovgu.featureide.visualisation" is redundant
#### Prerequisitives
* **FeatureIDE-Version:**  `3.6`

### Issue description
All classes and functionality of the Plug-In `de.ovgu.featureide.visualisation` are already present in `de.ovgu.featureide.ui`. Both add a menu entry. However, only the menu entry of the `de.ovgu.featureide.ui` works. Thus, we should remove `de.ovgu.featureide.visualisation`.

### PR DESCRIPTION
**Pull request by Subaro**
…on graphs. Complete functionality regarding the generation of feature relation graphs is outsourced into the fm.visualisation plugin.

### CODE REVIEW

**Comment by Subaro** [comment]
this pull request fixes #933

File: plugins/de.ovgu.featureide.visualisation/src/de/ovgu/featureide/visualisation/Utils.java [hunk 1]
```diff
@@ -36,46 +36,46 @@
 import org.osgi.framework.Bundle;
 
 /**
- * Utils for visualisations
```
Comment (PaulWestphal) [inline plugins/de.ovgu.featureide.visualisation/src/de/ovgu/featureide/visualisation/Utils.java]: why did you change the spelling to "visuali**z**ations" when we're now using "visuali**s**ations"?
